test/test_browser.py

```diff
@@ -3054,19 +3054,19 @@ def test_sdl2_image_formats(self):
   @no_wasm64('SDL2 + wasm64')
   def test_sdl2_key(self):
     create_file('pre.js', '''
-      function keydown(c) {
-        var event = new KeyboardEvent("keydown", { 'keyCode': c, 'charCode': c, 'view': window, 'bubbles': true, 'cancelable': true });
+      function keydown(keyCode, code) {
+        var event = new KeyboardEvent("keydown", { keyCode, code, charCode: keyCode, 'view': window, 'bubbles': true, 'cancelable': true });
```

Comment:
Does closure treat `keyCode` as a string identifier it will not minify even after this change?

It's no longer in quotes which is what worries me. Specifically I worry that minifying the parameters seems like it would lead to changing the string names:
```js
function a(b, c) {
  var d = new KeyboardEvent("keydown", { b, c, ..
``` 
Minifying `keyCode` to `b` leads to the object having `"b": b` which no longer works.

But perhaps closure is aware of this and will not minify the parameter names here? Or perhaps it switches from `{ b, c, ..` notation to `{ 'keyCode': b, 'code': c, ..` - ?

Reply:
I don't think we run this test with closure so its not important in this case.

Reply:
FWIW, I confirmed the closure seems to be aware of the various keys in the object.

````
$ cat test.js 
let foo = (code, keyCode) => {
  var e = new KeyboardEvent('keydown', { keyCode, code, charCode: keyCode, view: window, bubbles: true, cancelable: true, xxx: 1 });
  return e;
}
foo(1, 2);
$ ./emcc ~/test/hello.c --pre-js=test.js  --closure=1 --minify=0
$ grep "new Key" a.out.js 
new KeyboardEvent("keydown", {keyCode:2, code:1, charCode:2, view:window, bubbles:!0, cancelable:!0, h:1});                                                                                                       
```

Note that it minified my xxx key but not the others.

Reply:
I will create a followup PR to remove the unneeded key quotes.